### DISCUSSION TITLE
Hit circle dimming should not affect hidden

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20866] [Ideas] [answered]
I cannot remember what this feature is actually called so I cant search if this has already been suggested, I know the dim to full was added recently (hitcircles not in hit window are dimmed).

This should not affect hidden, as hit circles are barely even rendered in the hit window, so they will pretty much always be dimmed (making hidden unnecessarily darker.)

**Comment by peppy** [comment, marked as answer]
This would only be changed if it is to match stable.

**Reply by frenzibyte** [reply]
Checking with stable, I can't find any code that would remove dimming on hidden mod. And on visual comparison, I can't sense any differences as well:

https://user-images.githubusercontent.com/22781491/199590626-bf0c59b6-fa3c-4cc2-a174-f92c846597fc.mp4

https://user-images.githubusercontent.com/22781491/199590923-8d73fe8d-9c0f-4853-a730-0793cbb8c1af.mp4

**Reply by peppy** [reply]
Yeah, that was my expectation. So this is working as expected.
